### DISCUSSION TITLE
Is there a way to use RTSP as input and WebRTC as output?

### DISCUSSION THREAD

**Discussion by kimud6003** [bluenviron/mediamtx#3694] [Questions] [answered]
### Question

Hello,

I’m currently working on receiving RTSP streams from CCTV cameras and planning to use the MediaMTX API to input these RTSP URLs into the server. My goal is for the clients to be able to receive and play these RTSP streams via WebRTC.

However, I’ve noticed that the basic examples provided don’t exactly match the setup I’m trying to achieve.

What I’m looking to do is:

Configure the RTSP stream as an input to MediaMTX.
Allow clients to receive and play this RTSP stream in real-time through WebRTC.
Do you know of any examples or can you provide guidance on how to achieve this scenario? Any advice on configuration or API usage would be greatly appreciated.

Thank you!

**Comment by clee604** [comment, marked as answer]
I do this with my 'baby' cameras. I don't think I needed to do anything specific though. 

I set up to publish the stream proxy in the paths section of the mediamtx.yml
```
   wyzev2:
    source: rtsp://x.x.x.5:8554/1080p
    sourceOnDemand: yes
    sourceProtocol: tcp
```
Then I can access the webrtc stream using the mediamtx server address
`http://x.x.x.6:8889/wyzev2/`

**Reply by kimud6003** [reply]
Thank you. @clee604 
It works because of you.
However, webrtc videos are appearing every two seconds.
Was it shown on the screen in real time when rtsp was webrtc?

**Reply by kimud6003** [reply]
Oh, I've worked it out.
thanks

**Reply by prakasa1904** [reply]
Hi @kimud6003 , can you share complete config on this. I have same issue, to configure my child camera.. The webRTC not show any data in browser
